news/fix-input-output-docs.rst

```diff
@@ -0,0 +1,12 @@
+**Added:** * Add note that the target of redirection should be separated by a
```

Comment:
`**Added:**` should be on it's on line, then an empty line, then the bullet point